server.js

```diff
@@ -98,20 +104,26 @@ function join_custom_room(ws, join_room_json) {
     // Add a prefix to the room id to indicate custom match.
     room_id = "custom_" + room_id
 
+    // Extract room settings from the passed in json.
     var deck_id = join_room_json.deck_id
+    var starting_timer = join_room_json.starting_timer
```

Comment:
Can you do validation that these are all ints and also check if they're missing. if they fail either, can you set them to some good default values?